src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/Application.java

```diff
@@ -17,23 +20,38 @@
     public static final String APPLICATION_NAME = "chips-rest-interfaces-consumer";
 
     @Autowired
+    @Lazy
     @Qualifier("main-looping-consumer")
     private LoopingMessageProcessor loopingMainMessageConsumer;
 
     @Autowired
+    @Lazy
     @Qualifier("retry-looping-consumer")
     private LoopingMessageProcessor loopingRetryMessageConsumer;
 
+    @Autowired
+    private ApplicationLogger logger;
+
+    @Value("${IS_ERROR_CONSUMER:false}")
```

Comment:
I think the naming could be clearer - 
for config how about 'RUN_APP_IN_ERROR_MODE
and boolean would be isRunningInErrorMode